### ISSUE TITLE
Separation of thing and description and use of replacee to link to previous version

### ISSUE THREAD

**Issue by coret** [netwerk-digitaal-erfgoed/requirements-datasets#42]
A vendor proposes to keep alive previous version of datasetdescriptions (actually all metadata constructs). A changed datasetdescriptions gets a new @id The @id of the datasetdescription it is replacing is stored in the https://schema.org/replacee property. 

This vendor also distinghuises between a thing and its description. So the Dataset has a durable identifier and the description a version specific @id.

This separation and keeping/linking of old versions of the description seems clean, albeit a bit more complex at first sight. Can we incorporate this in our requirements and crawler (deleting of previous version based on replacee @id).

**Comment by EnnoMeijers** [comment]
This trigger for me the question if we should consider implementing the Memento protocol either at the source or on the level of the Datasetregister level so it supports browsing the history of dataset descriptions?

**Comment by ddeboer** [comment]
In my eyes, these are two separate questions:

1. Should the Register support historical versions of dataset descriptions?
2. Should the Register support a differentiation between dataset resources and dataset description resources?

In our discussion, @coret and I came up with the following provisional answers:

1. Only if there are relevant use cases for it. 3. If the source supports versioning, consumers (such as researchers) don’t need versioned data in the Register, because they can reference and cite historical versions in the original source. On the other hand, does the Knowledge Graph need historical data, for instance to analyse change frequency of dataset descriptions?
2. No, because the Register already defines datasets as their descriptions. Publishers are still free to maintain the distinction as long as they use [permanent IRIs](https://netwerk-digitaal-erfgoed.github.io/requirements-datasets/#identifiers) for the dataset descriptions in their catalogs.

**Comment by EnnoMeijers** [comment]
I agree with both points but in practice very little sources will be able to support versioning is my assumption. So adding this functionality using a Memento like solution would be a valuable addition for researchers and other consumers that require  versioned/timestamped datasets. Maybe add this to point to the backlog for further exploration?
